script/build

```diff
@@ -60,13 +60,15 @@ def main():
       elif component == 'static_library' and args.no_static_library:
         continue
       out_dir = get_output_dir(SOURCE_ROOT, target_arch, component)
-      target = 'chromiumcontent:chromiumcontent'
-      subprocess.check_call([NINJA, '-C', os.path.relpath(out_dir), target], env=env)
+      subenv = env.copy()
       if component == 'static_library':
-        subenv = env.copy()
         subenv['CHROMIUMCONTENT_2ND_PASS'] = '1'
         target = 'chromiumcontent:libs'
-        subprocess.check_call([NINJA, '-C', os.path.relpath(out_dir), target], env=subenv)
+      elif component == 'tests':
+        target = 'chromiumcontent:chromiumcontent_tests'
+      else:
+        target = 'chromiumcontent:chromiumcontent'
+      subprocess.check_call([NINJA, '-C', os.path.relpath(out_dir), target], env=subenv)
```

Comment:
We probably need to make `target` a build script argument to allow building and running tests
in a different build configuration, e.g. `shared_library`.

I can do it in a separate PR.